lib/board.py

```diff
@@ -1,9 +1,29 @@
 class Board:
-    board = {1: " ", 2: " ", 3: " ", 4: " ", 5: " ", 6: " ", 7: " ", 8: " ", 9: " "}
+    def __init__(self):
+        self.board = {
+            "1": " ",
+            "2": " ",
+            "3": " ",
+            "4": " ",
+            "5": " ",
+            "6": " ",
+            "7": " ",
+            "8": " ",
+            "9": " ",
+        }
 
-    def display_board():
-        print(Board.board[1] + "  | " + Board.board[2] + " | " + Board.board[3])
+    def display_board(self):
+        print(self.board["1"] + "  | " + self.board["2"] + " | " + self.board["3"])
         print("-----------")
-        print(Board.board[4] + "  | " + Board.board[5] + " | " + Board.board[6])
+        print(self.board["4"] + "  | " + self.board["5"] + " | " + self.board["6"])
         print("-----------")
-        print(Board.board[7] + "  | " + Board.board[8] + " | " + Board.board[9])
+        print(self.board["7"] + "  | " + self.board["8"] + " | " + self.board["9"])
+
+    def move(self, input, mark):
+        self.board[input] = mark
+
+    def position_taken(self, input):
+        return self.board[input] != " "
+
+    def board_full(self):
```

Comment:
You may remember in ruby that when a method returns a boolean value, it should stylistically be appended with a `?`, so for example it would be `board_full?`.

The ? is not a thing in python, but most often I have seen these methods styled with `is` or `has` at the beginning to note that it is an expected `true` or `false` return value. You don't have to change it here, but I would recommend looking into python naming conventions for this to decide how to keep them consistent.

Reply:
I surely will do it. Thank you!